assets/base.css

```diff
@@ -1445,7 +1445,15 @@ details[open] > .share-button__fallback {
   z-index: 3;
```

Comment:
Because we removed the corner radius on the bottom right from the cart notification, it can sometimes look like this: https://screenshot.click/05-50-wqgpv-vn6wh.png cc @melissaperreault 

I will let you test and see what you think of it with the different settings 🤔

Reply:
Yes, this was expected to keep visible and usable the drag handle UI. Thanks!